src/components/UiCardList/constants.ts

```
@@ -3,65 +1,20 @@
+import Diamond      from '../../features/landing/assets/svg/possibilities/diamond.svg';
+import Ruby         from '../../features/landing/assets/svg/possibilities/ruby.svg';
 import SmallDiamond from '../../features/landing/assets/svg/possibilities/smallDiamond.svg';
-import SmallRuby from '../../features/landing/assets/svg/possibilities/smallRuby.svg';
-import WhyUsCodeIcon from '../../features/landing/assets/svg/why-us/code.svg';
-import WhyUsIntegrationsIcon from '../../features/landing/assets/svg/why-us/integrations.svg';
-import WhyUsMigrationIcon from '../../features/landing/assets/svg/why-us/migration.svg';
-import WhyUsServicesIcon from '../../features/landing/assets/svg/why-us/services.svg';
-import WhyUsSettingsIcon from '../../features/landing/assets/svg/why-us/settings.svg';
-import WhyUsTemplatesIcon from '../../features/landing/assets/svg/why-us/templates.svg';
+import SmallRuby    from '../../features/landing/assets/svg/possibilities/smallRuby.svg';
```

ESLint

[error] There should be no empty line within import group

(import/order)